### PR TITLE
fix: navbar

### PR DESCRIPTION
**Pull request by sk-tenba**
https://app.asana.com/0/home/1203849283056222/1204278549695437

### CODE REVIEW

**Comment by sk-tenba** [comment]
[Screencast from 28-03-2023 16:11:42.webm](https://user-images.githubusercontent.com/110231356/228188542-265b4c80-02ad-487d-a8e9-fea5d1f9f3f2.webm)

File: containers/LandingPage/containers/NavBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        }
+        className={styles.rightColumn}
+        ref={setRightColumnContainer}
+      >
+        <Link href="/gallery">
+          <div className={styles.buttonNav}>Gallery</div>
+        </Link>
+        <Link href="#reward">
+          <div className={styles.buttonNav}>KreAtaveRse Story</div>
+        </Link>
```
Comment (sk-kitsune) [inline containers/LandingPage/containers/NavBar/index.tsx:58]: label "KreAtaveRse Story" but points to "#reward" ???

Comment (sk-kitsune) [reply containers/LandingPage/containers/NavBar/index.tsx:58]: 🔴 **Blocking issue**: this PR cannot be merged until this issue is resolved.

File: containers/LandingPage/containers/NavBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          <div className={styles.buttonNav}>
+            Start Kolouring THE KREATAVERSE
+          </div>
+        </Link>
+        <Link href="#reward">
+          <div className={styles.buttonNav}>your rewards</div>
+        </Link>
+        <Link href="#roadmap">
+          <div className={styles.buttonNav}>ROADMAP</div>
+        </Link>
```
Comment (sk-kitsune) [inline containers/LandingPage/containers/NavBar/index.tsx:69]:
```
Gallery
KreAtaveRse Story
Start Kolouring THE KREATAVERSE
your rewards
ROADMAP
```


please be consistent

**Review by sk-kitsune — DISMISSED** [review]
✅ **Approved**: no blocking issues

**Pull request merged**